src/SWP/Bundle/MultiTenancyBundle/Resources/config/services.yml

```diff
@@ -17,7 +17,6 @@ services:
     swp_multi_tenancy.tenant_resolver:
         class: '%swp_multi_tenancy.tenant_resolver.class%'
         arguments:
-            - '%domain%'
```

Comment:
if it is not used anymore it should be removed also from the parameters.yml.dist